templates/ocs-deployment/rbd_storage_class.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+   name: csi-rbd
+provisioner: rbd.csi.ceph.com
+parameters:
+    # Comma separated list of Ceph monitors
+    # if using FQDN, make sure csi plugin's dns policy is appropriate.
+    monitors: {{ mon_ports }}
+
+    # if "monitors" parameter is not set, driver to get monitors from same
+    # secret as admin/user credentials. "monValueFromSecret" provides the
+    # key in the secret whose value is the mons
+    #monValueFromSecret: "monitors"
+    
+    # Ceph pool into which the RBD image shall be created
+    pool: {{ pool_name }}
+
+    # RBD image format. Defaults to "2".
+    imageFormat: "2"
+
+    # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
+    imageFeatures: layering
+    
+    # The secrets have to contain Ceph admin credentials.
+    csi.storage.k8s.io/provisioner-secret-name: {{ ceph_secret }}
```

Comment:
Hi Warren,

The rook-csi documentation mentions creation of an oc secret using the secret.yaml file from [rook-csi repo](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/csi/example/rbd/secret.yaml). 

So, are we not going to make a jinja2 template for secret.yaml and then create a yaml out of it(similar to storageclass.yaml ) OR are we going to follow this **utility/get_ceph_secret.py** process to get secret ?

Reply:
Well I am confused too.  If your pull request for CSI templates for rbd and cephfs allow one to create a pvc for a ceph rbd pool, then I think that this pr is a duplicate of what you are doing.  I will hold off on doing anything here for the moment.